apps/aecore/test/aec_mining_tests.erl

```diff
@@ -40,10 +40,10 @@ mine_block_test_() ->
                  %% let_it_crash = generate_valid_test_data(TopBlock, 100000000000000),
                  Nonce = case aec_hard_forks:protocol_effective_at_height(Height + 1) of
                              ?ROMA_PROTOCOL_VSN    -> 1157794539819639234;
-                             ?MINERVA_PROTOCOL_VSN -> 9186874827433957562;
-                             ?FORTUNA_PROTOCOL_VSN -> 7910161413761453387;
+                             ?MINERVA_PROTOCOL_VSN -> 14355842201894252133;
+                             ?FORTUNA_PROTOCOL_VSN -> 7559967028720457330;
```

Comment:
Could you remind me why do we change these?

Reply:
Same reason as [last time](https://github.com/aeternity/aeternity/pull/3450#discussion_r553826166). Since we change the block header info, the tests that expect certain nonce to work suddenly fail :)